defaults/manager_restore_benchmark_snapshots.yaml

```diff
@@ -8,7 +8,7 @@ sizes:  # size of backed up dataset in GB
         - standard1: 1
     number_of_rows: 1073760
     exp_timeout: 1200  # 20 minutes (timeout for restore data operation)
-    scylla_version: "2024.2.0-rc0"
+    scylla_version: "2024.2.0-rc1"
```

Comment:
rc3 is available

Reply:
Yeah, but we are going to wait for official release and rebuild the snapshots.
The change here is typing fix as initially the version `...~rc0` was put there wrongly.